features/step_definitions/product.rb

```diff
@@ -138,38 +139,38 @@ def build_product
 
 When(/^the user creates a product with multiple measures from different groups$/) do
   steps %( When the user fills out all product information but measures )
-  page.all('#measure_tabs .ui-tabs-nav a')[2].click
+  page.all('#measure_tabs .ui-tabs-nav a')[0].click
   page.all('input.measure-checkbox')[0].click
-  page.all('#measure_tabs .ui-tabs-nav a')[3].click
+  page.all('#measure_tabs .ui-tabs-nav a')[1].click
   page.all('input.measure-checkbox')[0].click
   page.click_button 'Add Product'
 end
 
 When(/^the user creates a product with selecting a group of measures$/) do
   steps %( When the user fills out all product information but measures )
-  page.find("[href='#Miscellaneous_div']").click
+  # page.find("[href='#Miscellaneous_div']").click
```

Comment:
If these lines aren't necessary, should they be removed rather than commented out?